core/sandbox/package.json

```diff
@@ -32,7 +32,8 @@
   },
   "dependencies": {
     "@dotcom-tool-kit/next-router": "file:../../plugins/next-router",
-    "@dotcom-tool-kit/node": "file:../../plugins/node"
+    "@dotcom-tool-kit/nodemon": "file:../../plugins/nodemon",
+    "nodemon": "^2.0.15"
```

Comment:
**nitpick (super inconsequential)**: these should all be `devDependencies`. doesn't matter at all but might be slightly confusing if anybody uses the sandbox as a reference